src/Tickets/Commerce/Gateways/Square/REST/Webhook_Endpoint.php

```diff
@@ -0,0 +1,378 @@
+<?php
+
+/**
+ * Square Webhook Endpoint
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Gateways\Square\REST
+ */
+
+namespace TEC\Tickets\Commerce\Gateways\Square\REST;
+
+use TEC\Tickets\Commerce\Gateways\Contracts\Abstract_REST_Endpoint;
+use TEC\Tickets\Commerce\Gateways\Square\Webhooks;
+use TEC\Tickets\Commerce\Gateways\Square\Order;
+use WP_REST_Request;
+use WP_REST_Server;
+use WP_REST_Response;
+use WP_Error;
+
+/**
+ * Class Webhook_Endpoint.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Gateways\Square\REST
+ */
+class Webhook_Endpoint extends Abstract_REST_Endpoint {
+
+	/**
+	 * The REST namespace for this endpoint.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected string $namespace = 'tribe/tickets/v1';
+
+	/**
+	 * The REST endpoint path for this endpoint.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected string $path = '/commerce/square/webhooks';
```

Comment:
Could those be constants instead ?

Reply:
Maybe we can change this for all webhooks. But let me do that on a separate PR.